generated/niscope/session.py

```diff
@@ -2509,12 +2519,18 @@ def fetch_measurement_stats(self, scalar_meas_function, timeout=5.0):
         channel_list_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
         timeout_ctype = visatype.ViReal64(timeout)  # case S150
         scalar_meas_function_ctype = visatype.ViInt32(scalar_meas_function.value)  # case S130
-        result_ctype = (visatype.ViReal64 * self._actual_num_wfms())()  # case B560
-        mean_ctype = (visatype.ViReal64 * self._actual_num_wfms())()  # case B560
-        stdev_ctype = (visatype.ViReal64 * self._actual_num_wfms())()  # case B560
-        min_ctype = (visatype.ViReal64 * self._actual_num_wfms())()  # case B560
-        max_ctype = (visatype.ViReal64 * self._actual_num_wfms())()  # case B560
-        num_in_stats_ctype = (visatype.ViInt32 * self._actual_num_wfms())()  # case B560
+        result_size = self._actual_num_wfms()  # case B560
```

Comment:
I guess we still call _actual_num_wfms() a bunch of times :(

Reply:
Yes, those are all separate variables and there is no lookahead or lookbehind when coming up with those declarations.